app.js

```diff
@@ -1,44 +1,102 @@
-var express = require('express');
-var path = require('path');
-var favicon = require('serve-favicon');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
+// Get dependencies
+const express = require('express');
+const path = require('path');
+const logger = require('morgan');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
+const http = require('http');
+const passport = require('passport');
+const flash = require('connect-flash');
+const session = require('express-session');
+const async = require('async');
+const nodemailer = require('nodemailer');
+const bcrypt = require('bcrypt-nodejs');
```

Comment:
Its not being used for now, but will be used if the password is encrypted and stored in the database  and this library helps in decrypting the stored password and comparing it with the password the user uses to login. I added it for future use. Will remove it now and re-add it later.